src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -140,11 +140,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
+    Boolean isdelete = Boolean.valueOf(request.getParameter("delete"));
```

Comment:
Rename to shouldDelete

Make sure your variable names are grammatically correct / camel case